docs/maintaining.md

```diff
@@ -4,11 +4,72 @@ This document is for people working on VTEX as Toolbelt maintainers.
 
 **Table of Contents**
 
+- [Deploying a new version](#deploying-a-new-version)
 - [Forcing a new version](#forcing-a-new-version)
 - [Deprecating a version](#deprecating-a-version)
 - [Changing the release message](#changing-the-release-message)
 - [Monitoring and debugging user errors](#monitoring-and-debugging-user-errors)
 
+### Deploying a new version
+
+#### **NPM**
+
+The main deploy of `toolbelt` is on [NPM](https://www.npmjs.com/package/vtex). This deploy is reponsable for warning `new and deprecated versions`
+
+Here's how you deploy on `NPM`:
+
+- Fire this command on terminal in `toolbelt` project root path (*Stable version*):
+
+```bash
+    releasy patch --stable
+    releasy minor --stable
+    releasy major --stable
+```
+
+- Fire this command on terminal in `toolbelt` project root path (*Beta version*):
+
+```bash
+    releasy patch
+    releasy minor
+    releasy major
+```
+
+This will release a `github tag` and trigger the `npm-publish github action`
+
+#### **AWS S3**
+
+This deploy will contain the `toolbelt standalone tarball` that is compatible with `MacOS` and `Linux`
+
+Here's how you deploy on `AWS S3`: 
+
+- First set your `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` env variables, this keys values is under the possession and control of `@vtex/io-devtools`.
+
+- *Ps*: You might have `7-zip` installed on your machine.
```

Comment:
Suggest you to add the commands to install:
```
brew update
 brew install p7zip
```

Reply:
I don't think that this command is needed since you can deploy from any `OS`, and also you can install `7-zip` from different ways. What do you think ?

Reply:
Makes sense